manual/README.md

```diff
@@ -15,16 +15,19 @@ Note that you must make sure `hevea.sty` is installed into TeX properly. Your
 package manager may not do this for you. Run `kpsewhich hevea.sty` to check.
 
 
-Building
+Building the documentaiton
```

Comment:
s/documentaiton/documentation

Reply:
Actually "manual" would make slightly more sense than "documentation" here, wouldn't it?